radis/test/misc/test_utils.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import print_function, absolute_import, division, unicode_literals
+import pytest
+
+
+def test_files_from_regex(*args, **kwargs):
+    from radis.misc.utils import get_files_from_regex
+    from radis.test.utils import getTestFile
+
+    path = "/home/aditya/radis/radis/test/files/*.par"
```

Comment:
this should use getTestFile ?

Reply:
This is just so that I can quickly generate a path to the .par files (as an test/example) anywhere this code runs.